lxd/api_1.0.go

```diff
@@ -777,6 +778,8 @@ func doApi10UpdateTriggers(d *Daemon, nodeChanged, clusterChanged map[string]str
 			fallthrough
 		case "loki.types":
 			lokiChanged = true
+		case "acme.domain":
```

Comment:
This should also be changed on `acme.ca_url` change right?

Reply:
We could do that.